### PR TITLE
Homepage refinements

### PR DESCRIPTION
**Pull request by jamiemccue**
Align the diagrams in the three columns on the homepage

Screen Shot 2019-06-06 at 11 53 53 AM

### CODE REVIEW

File: docs/can-canjs/canjs.md [hunk 1]
```diff
@@ -676,7 +673,7 @@ Below is an entire app that shows off some of the best features of CanJS:
 - `isPending`, `isRejected`, `isResolved`, and `value` helpers for directly reading the state of a Promise.
 - When you add a to-do, it automatically gets inserted into the list in the right position.
 
-<p class="codepen" data-height="530" data-theme-id="0" data-default-tab="js,result" data-user="bitovi" data-slug-hash="omqyMw" style="height: 530px; box-sizing: border-box; display: flex; align-items: center; justify-content: center; border: 2px solid black; margin: 1em 0; padding: 1em;" data-pen-title="CanJS 5 — Basic Todo App">
+<p class="codepen" data-height="580" data-theme-id="0" data-default-tab="js,result" data-user="bitovi" data-slug-hash="omqyMw" style="height: 530px; box-sizing: border-box; display: flex; align-items: center; justify-content: center; border: 2px solid black; margin: 1em 0; padding: 1em;" data-pen-title="CanJS 5 — Basic Todo App">
```
Comment (chasenlehara) [inline docs/can-canjs/canjs.md]: Was 580 the smallest it could be set to in order to avoid a scrollbar?

File: docs/can-canjs/canjs.md [hunk 2]
```diff
@@ -525,7 +528,7 @@ pre[class*=language-].line-numbers.line-numbers code {
           <a class="github-button nav-social" href="https://github.com/canjs/canjs" data-count-href="/canjs/canjs/stargazers" data-show-count="true">Star</a>
         </li>
         <li>
-          <a href="https://twitter.com/canjs" class="twitter-follow-button nav-social" data-show-count="true" data-show-screen-name="false">Follow @canjs</a><script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+          <a href="https://twitter.com/canjs" class="twitter-follow-button nav-social" data-show-count="true" data-show-screen-name="false">@canjs</a><script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
```
Comment (chasenlehara) [inline docs/can-canjs/canjs.md]: Any reason for this change?

File: docs/can-canjs/canjs.md [hunk 3]
```diff
@@ -458,7 +461,7 @@ pre[class*=language-].line-numbers.line-numbers code {
       <a href="./doc/guides/crud-beginner.html" class="btn">Learn how to build this CRUD app</a>
     </div>
     <div class="right-col">
-      <img src="docs/images/hero-image.svg" />
+      <img src="docs/images/hero-image.svg" width="642" />
```
Comment (chasenlehara) [inline docs/can-canjs/canjs.md]: We should set a height on this, right?

File: docs/can-canjs/canjs.md [hunk 4]
```diff
@@ -437,6 +437,9 @@ pre[class*=language-].line-numbers.line-numbers code {
   }
 }
 @@media (max-width: 529px) {
+  .hero-logo {
+    width: 260px;
+  }
```
Comment (chasenlehara) [inline docs/can-canjs/canjs.md]: Should this not be set on the SVG since the height is set on it?

Comment (jamiemccue) [reply docs/can-canjs/canjs.md]: This reduces the width of the logo at the smallest mobile dimension.

Comment (chasenlehara) [reply docs/can-canjs/canjs.md]: Is the aspect ratio correct even if the height isn’t set as well?

Comment (jamiemccue) [reply docs/can-canjs/canjs.md]: Yes, the aspect ratio is set by the width. The height as been removed.

**Comment by jamiemccue** [comment]
Fixed the jump in the hero section on load:

loading

**Comment by chasenlehara** [comment]
I can’t really tell how the images in the three columns are supposed to be aligned… the first two seem to be top aligned, but the third one kinda sits in the middle:
Screen Shot 2019-06-07 at 6 50 56 PM

…and at smaller widths, it doesn’t look like they’re really aligned:
Screen Shot 2019-06-07 at 6 50 35 PM

**Pull request merged**